modules/admin_manual/pages/configuration/user/user_provisioning_api.adoc

```diff
@@ -51,18 +70,41 @@ sending a basic HTTP authentication header.
 * 103 - unknown error occurred whilst adding the user
 * 104 - group does not exist
 
-==== Add User Example
+==== Usage Examples
 
+.Create the user "Frank" with password "frankspassword".
 [source,console]
 ----
-# Creates the user "Frank" with password "frankspassword"
 curl -X POST http://admin:secret@example.com/ocs/v1.php/cloud/users \
    -d userid="Frank" \
    -d password="frankspassword"
+----
 
-# Creates the user "Frank" with password "frankspassword" and adds him to the "finance" and "management" groups
+.Create the user "Frank" with password "frankspassword" and add him to the "finance" and "management" groups.
+[source,console]
+----
+curl -X POST http://admin:secret@example.com/ocs/v1.php/cloud/users \
+   -d userid="Frank" \
+   -d password="frankspassword" \
+   -d groups[]="finance" -d groups[]="management"
+----
+
+.Create the user "Frank" with password "frankspassword", email address "frank@example.com" and add him to the "finance" and "management" groups. The user will be emailed with a link to change the password.
+[source,console]
+----
+curl -X POST http://admin:secret@example.com/ocs/v1.php/cloud/users \
+   -d userid="Frank" \
+   -d email="frank@example.com" \
+   -d password="frankspassword" \
+   -d groups[]="finance" -d groups[]="management"
+----
+
+.Create the user "Frank" with email address "frank@example.com" and add him to the "finance" and "management" groups. A temporary password will be created and the user will be emailed with a link to change the password.
```

Comment:
Doesn't the previous example and this one look same? At least from the behaviour wise. If you agree with me then just remove the `-d password` line, then it would add more info. It would help users know that without providing password, the user can be created and with the help of email user can set the password.

Reply:
Thanks for drawing my attention to this. I'll check it out.